app/controllers/grants/grant_users_controller.rb

```diff
@@ -89,10 +90,15 @@ def set_grant_user
       @grant_user = GrantUser.find(params[:id])
     end
 
-    def authorize_grant
+    def authorize_grant_viewer_access
```

Comment:
grant_user will always be a nested resource off of grant and grant will always need to be defined. Using that policy and those specific methods to authorize against seems sufficient. Especially since any grant_user policy would just be calling the methods in the grant.